charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -1482,13 +1482,21 @@ def send_request_if_needed(request, relation='ceph'):
             relation_set(relation_id=rid, broker_req=request.request)
 
 
+def has_broker_rsp(rid=None, unit=None):
+    rdata = relation_get(rid=rid, unit=unit) or {}
```

Comment:
This function needs a docstring please, something along the lines of:

```python
"""Return True if the broker_rsp key is 'truthy' (i.e. set to semeting) in the relation data.

:param rid: The relation to check (default of None means current relation)
:type rid: Union[str, None]
:param unit: The remote unit to check (default of None means current unit)
:type unit: Union[str, None]
:returns: True if broker key exists and is set to something 'truthy'
:rtype: bool
"""
```

Reply:
Thanks